cg/models/taxprofiler/taxprofiler.py

```diff
@@ -30,3 +32,33 @@ class TaxprofilerSample(NextflowSample):
     """Taxprofiler sample model is used when building the sample sheet."""
 
     instrument_platform: SequencingPlatform
+    run_accession: str
+    fasta: str
+
+    @staticmethod
+    def headers() -> List[str]:
+        """Return sample sheet headers."""
+        return [
+            "sample",
```

Comment:
These hard-coded values can be replaced by constants, but that could be done in another PR